cmd/commands/subvolume.go

```diff
@@ -40,22 +40,23 @@ var listCmd = &cobra.Command{
 }
 
 var deleteCmd = &cobra.Command{
-	Use:                "delete",
-	Short:              "Deletes a stale subvolume.",
-	DisableFlagParsing: true,
-	Args:               cobra.ExactArgs(3),
-	Example:            "kubectl rook-ceph delete <subvolumes> <filesystem> <subvolumegroup>",
+	Use:     "delete",
+	Short:   "Deletes a stale subvolume.",
+	Args:    cobra.ExactArgs(3),
+	Example: "kubectl rook-ceph delete <subvolumes> <filesystem> <subvolumegroup>",
```

Comment:
You need to swap the filesystem and subvolume, IMHO allowing one at a time also fine.

Reply:
ohh yeah, forgot to change here.